app/src/test/java/com/manuel28g/carsales/covidworlddata/viewmodel/CovidInfoViewModelTest.kt

```diff
@@ -0,0 +1,192 @@
+package com.manuel28g.carsales.covidworlddata.viewmodel
+
+import androidx.arch.core.executor.testing.InstantTaskExecutorRule
+
+import com.manuel28g.carsales.covidworlddata.model.CovidInfo
+import com.manuel28g.carsales.covidworlddata.model.CovidInfoStatus
+import com.manuel28g.carsales.covidworlddata.repository.CovidData
+
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.flow.flow
+import kotlinx.coroutines.test.TestCoroutineDispatcher
+import kotlinx.coroutines.test.runBlockingTest
+import kotlinx.coroutines.test.setMain
+
+import org.junit.Assert.*
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+
+import org.mockito.Mock
+import org.mockito.kotlin.mock
+import org.mockito.kotlin.whenever
+import java.lang.IllegalStateException
+import java.lang.NullPointerException
+
+import java.text.SimpleDateFormat
+
+import java.util.*
+
+@RunWith(JUnit4::class)
+class CovidInfoViewModelTest {
+
+    @Rule @JvmField
+    val instantTaskExecutorRule = InstantTaskExecutorRule()
+    lateinit var viewModel:CovidInfoViewModel
+    private val mDayInMillis = 1000*60*60*24L
+
+    @Mock
+    lateinit var repository: CovidData
+
+    private fun getBaseResponse():CovidInfo{
```

Comment:
You can have another class (that can be an Object type of class) that returns you this CovidInfo object. This way this test class is cleaner 👍